contracts/Coordinator.sol

```diff
@@ -38,16 +41,14 @@ contract Coordinator is ICoordinator, AccessController {
         _;
     }
 
-    constructor(address admin) AccessController(admin) {}
-
-    function init(
+    function setDependencies(
```

Comment:
we arent using the "initialize()" pattern?

Reply:
We are using initialize() pattern but keeping the parameters of it to a minimum. When we deploy contracts, initialize() is the first thing that runs and is the constructor in a way. setDependencies is an admin action we'll run after we deploy. 

Makes deploying contracts simpler and less error prone and gives us more flexibility with Dependencies